pro/src/pages/OfferIndividualWizard/Confirmation/__specs__/Confirmation.spec.tsx

```diff
@@ -117,18 +124,13 @@ describe('Summary', () => {
       screen.getByText('Créer une nouvelle offre', { selector: 'a' })
     ).toHaveAttribute(
       'href',
-      generatePath(
-        getOfferIndividualPath({
-          step: OFFER_WIZARD_STEP_IDS.INFORMATIONS,
-          mode: OFFER_WIZARD_MODE.CREATION,
-          isCreation: true,
-        }) + '?structure=OFID&lieu=VID physical'
-      )
+      generatePath('/offre/creation' + '?structure=OFID&lieu=VID physical')
     )
     expect(
       screen.getByText('Voir la liste des offres', { selector: 'a' })
     ).toHaveAttribute('href', `/offres`)
   })
+
```

Comment:
Des lignes vides entre les `it` => une personne de qualité :heart: (ou bien psychorigide du formattage comme moi ahah)

Reply:
je trouve ça plus facile à lire haha